app/components/Classifier.tsx

```diff
@@ -13,16 +13,32 @@ function runModelProcess(baseArgs: string[]): IPty {
   const isWin = !isDev && process.platform === 'win32';
   const isLinux = !isDev && process.platform === 'linux';
 
-  const options = { cwd: 'models_runner' };
+  const modelName = 'serengeti';
+  const linuxBaseModelPath = `../${modelName}/model/trained_model.pkl`;
```

Comment:
Could we use [`path.join()`](https://nodejs.org/api/path.html#path_path_join_paths) to handle slash differences in Windows and POSIX paths? With `const modelsRoot = '..'` we could build paths like `path.join(modelsRoot, modelName, 'model', 'trained_model.pkl')`.

BTW, could we just put `trained_model.pkl` in its parent directory instead of nesting it as `model/trained_model.pkl`?